_tests/integration/manual_config_test.go

```diff
@@ -449,6 +449,11 @@ var customConfigResultYML = fmt.Sprintf(`options:
                 config: default-macos-config
               none:
                 config: default-macos-config
+  node-js: 
+	title: Project Root
+	summary: The directory containing the package.json file
+	env_key: PACKAGE_JSON_DIR
```

Comment:
This is a yaml string and it seems like these lines have more white spaces than the type. I think this might cause some issues when parsing it.